lab_3_ann_retriever/main.py

```diff
@@ -130,6 +156,9 @@ def _remove_stop_words(self, tokens: list[str]) -> list[str] | None:
 
         In case of corrupt input arguments, None is returned.
         """
+        if not tokens or not isinstance(tokens, list):
+            return None
+        return list(word for word in tokens if not word in self._stop_words)
```

Comment:
can we create a list right away as list comprehension?

Reply:
done!